src/components/organisms/pupil/OakHintButton/OakHintButton.tsx

```diff
@@ -51,6 +51,8 @@ export const OakHintButton = (props: OakHintButtonProps) => {
       disabled={props.disabled}
       iconBackgroundSize={"all-spacing-8"}
       iconSize={"all-spacing-6"}
+      data-rac
+      aria-describedby="hint-tooltip"
```

Comment:
I think this `id` should be passed in as a prop because it is nested in OakQuizHint.tsx and relies on the same id.

Alternatively allow all the properties of the button to be overridden 
```buttonProps?: Partial<InternalShadowRoundButtonProps & HTMLAttributes<Element>>;```
then
```{...buttonProps}``` on the StyledInternalShadowRoundButton

and use like

```<OakHintButtonProps buttonProps={{"aria-describedby":"hint-tooltip"}} />```